File: java/src/main/java/br/com/vps/consulting/b2b/management/partner/infrastructure/persistence/jpa/PartnerJpaRepository.java
package br.com.vps.consulting.b2b.management.partner.infrastructure.persistence.jpa;

import br.com.vps.consulting.b2b.management.partner.infrastructure.persistence.PartnerEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface PartnerJpaRepository extends JpaRepository<PartnerEntity, UUID> {
}

File: kotlin/src/main/kotlin/br/com/vps/consulting/b2b/management/partner/infrastructure/persistence/jpa/PartnerJpaRepository.kt
package br.com.vps.consulting.b2b.management.partner.infrastructure.persistence.jpa

import br.com.vps.consulting.b2b.management.partner.infrastructure.persistence.PartnerEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.jpa.repository.JpaSpecificationExecutor
import java.util.*

interface PartnerJpaRepository : JpaRepository<PartnerEntity, UUID>, JpaSpecificationExecutor<PartnerEntity>
